gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/jsr223/CoreImports.java

```diff
@@ -260,6 +261,7 @@
         CLASS_IMPORTS.add(PeerPressureVertexProgram.class);
         CLASS_IMPORTS.add(PageRankMapReduce.class);
         CLASS_IMPORTS.add(PageRankVertexProgram.class);
+        CLASS_IMPORTS.add(PageRankVertexProgramStep.PageRank.class);
```

Comment:
I think `PeerPressureVertexProgramStep.PeerPressure` should be a core import too.